tests/test_client/tests.py

```diff
@@ -85,6 +87,83 @@ def test_post(self):
         self.assertEqual(response.templates[0].name, 'POST Template')
         self.assertContains(response, 'Data received')
 
+    def test_json_post(self):
+        """ POST some JSON data to a view """
+        post_data = {
+            'value': 37
+        }
```

Comment:
This (and similar data dicts below) can be single lined.

Reply:
copy paste from similar test, will reformat.